python/segyio/segy.py

```diff
@@ -28,7 +28,7 @@
 
 class SegyFile(object):
 
-    def __init__(self, filename, mode, iline=189, xline=193, t0=1111.0):
+    def __init__(self, filename, mode, iline=189, xline=193):
```

Comment:
I think that it should be possible to set `t0` unless it is trivial and guaranteed to get that value from the headers. The default should probably be `none` to allow to find in headers and default to `0`, while a value of `0` is specific.